services.yaml

```diff
@@ -909,49 +926,197 @@ paths:
           description: NOT FOUND
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+    get:
+      tags:
+        - View Folders
+        - VFs
+      summary: "List View Folders"
+      description: "List View Folders"
+      responses:
+        200:
+          description: List of VF names
+          schema:
+            properties:
+              vf_names:
+                type: "array"
+                items:
+                  type: "object"
+                  properties:
+                    name:
+                      $ref: "#/definitions/Folder"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders/{folder_id}/views:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+      - in: path
+        name: folder_id
+        description: View folder to use
+        type: string
+        required: true
+    post:
+      tags:
+        - View Folders
+        - VFs
+      summary: "Add view to selected folder"
+      description: "Add view to selected folder"
+      parameters:
+        - in: body
+          description: View ID (string)
+          schema:
+            name: view_id
+            type: String
+            example: P12
+          required: true
+      responses:
+        201:
```

Comment:
should return 200, as there's no URI created (201 is when you are actually introducing a new URI that can be used, and then the 201 should return it as Location header)

Reply:
fixed